apps/a11y-tests/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "a11y-tests",
-  "version": "7.0.0",
+  "version": "8.0.0-beta",
```

Comment:
For the unpublished packages do we even need to bother with the -beta, or can we just put them straight to 8? Or possibly take back to 1 if that makes more sense.

Reply:
i intended to have `8.0.0-beta` everywhere without any `8.0.0` to avoid confusion. i think i will go with reseting `private` package version to `1.0.0` as you suggested :)